<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>397</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="newButton">
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="previewGroupBox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>381</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>PreviewArea</string>
    </property>
    <widget class="QPushButton" name="playPreviewButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>341</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Play Preview</string>
     </property>
    </widget>
    <widget class="QSlider" name="fpsSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>60</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="fpsSpinBox">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>290</y>
       <width>45</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>60</number>
     </property>
    </widget>
    <widget class="QLabel" name="previewFrame">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>256</width>
       <height>256</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="scaleToggle">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>360</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale-up</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="loopToggle">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>360</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Loop</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="addFrame">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>590</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Frame</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="frameNumber">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame Number:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>411</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Create A New Sprite</string>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="sizeLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>71</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Canvas Size</string>
     </property>
    </widget>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>166</width>
       <height>27</height>
      </rect>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </widget>
   <widget class="FrameEditor" name="frameEditor" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>580</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Color:</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentColor">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>600</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteFrame">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>590</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Frame</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionNew"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionReadMe"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuTools">
     <property name="title">
      <string>Tools</string>
     </property>
     <widget class="QMenu" name="menuShapes">
      <property name="title">
       <string>Shapes</string>
      </property>
      <addaction name="actionCircle"/>
      <addaction name="actionSquare"/>
      <addaction name="actionRectangle"/>
     </widget>
     <addaction name="actionEyedrop_Tool"/>
     <addaction name="actionFill"/>
     <addaction name="actionFill_All"/>
     <addaction name="actionBrush"/>
     <addaction name="actionEraser"/>
     <addaction name="actionMirror"/>
     <addaction name="menuShapes"/>
    </widget>
    <addaction name="actionColor_Picker"/>
    <addaction name="menuTools"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBrush"/>
   <addaction name="separator"/>
   <addaction name="actionEraser"/>
   <addaction name="separator"/>
   <addaction name="actionMirror"/>
   <addaction name="separator"/>
   <addaction name="actionColor_Picker"/>
   <addaction name="separator"/>
   <addaction name="actionEyedrop_Tool"/>
   <addaction name="separator"/>
   <addaction name="actionFill"/>
   <addaction name="separator"/>
   <addaction name="actionFill_All"/>
   <addaction name="separator"/>
   <addaction name="actionShapes"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="secondaryToolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionCircle"/>
   <addaction name="actionSquare"/>
   <addaction name="actionRectangle"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="iconResources.qrc">
     <normaloff>:/images/Images/open-folder-outline-icon.png</normaloff>:/images/Images/open-folder-outline-icon.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="iconResources.qrc">
     <normaloff>:/images/Images/save-outline-icon.png</normaloff>:/images/Images/save-outline-icon.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="iconResources.qrc">
     <normaloff>:/images/Images/mathematics-sign-plus-outline-icon.png</normaloff>:/images/Images/mathematics-sign-plus-outline-icon.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionColor_Picker">
   <property name="icon">
    <iconset resource="iconResources.qrc">
     <normaloff>:/images/Images/painting-icon.png</normaloff>:/images/Images/painting-icon.png</iconset>
   </property>
   <property name="text">
    <string>Color Picker</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionEyedrop_Tool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="iconResources.qrc">
     <normaloff>:/images/Images/dosage-icon.png</normaloff>:/images/Images/dosage-icon.png</iconset>
   </property>
   <property name="text">
    <string>Eyedrop Tool</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="actionFill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="iconResources.qrc">
     <normaloff>:/images/Images/painting-bucket-logo-icon.png</normaloff>:/images/Images/painting-bucket-logo-icon.png</iconset>
   </property>
   <property name="text">
    <string>Fill</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionFill_All">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="iconResources.qrc">
     <normaloff>:/images/Images/paint-bucket-icon.png</normaloff>:/images/Images/paint-bucket-icon.png</iconset>
   </property>
   <property name="text">
    <string>Fill All</string>
   </property>
   <property name="shortcut">
    <string>Shift+F</string>
   </property>
  </action>
  <action name="actionShapes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="iconResources.qrc">
     <normaloff>:/images/Images/square.png</normaloff>:/images/Images/square.png</iconset>
   </property>
   <property name="text">
    <string>Shapes</string>
   </property>
  </action>
  <action name="actionBrush">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="iconResources.qrc">
     <normaloff>:/images/Images/art-brush-design-icon.png</normaloff>:/images/Images/art-brush-design-icon.png</iconset>
   </property>
   <property name="text">
    <string>Brush</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="iconResources.qrc">
     <normaloff>:/images/Images/eraser-icon.png</normaloff>:/images/Images/eraser-icon.png</iconset>
   </property>
   <property name="text">
    <string>Eraser</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionMirror">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="iconResources.qrc">
     <normaloff>:/images/Images/flip-horizontal-icon.png</normaloff>:/images/Images/flip-horizontal-icon.png</iconset>
   </property>
   <property name="text">
    <string>Mirror</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="iconResources.qrc">
     <normaloff>:/images/Images/circle.png</normaloff>:/images/Images/circle.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionSquare">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="iconResources.qrc">
     <normaloff>:/images/Images/square.png</normaloff>:/images/Images/square.png</iconset>
   </property>
   <property name="text">
    <string>Square</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="iconResources.qrc">
     <normaloff>:/images/Images/rectangle.png</normaloff>:/images/Images/rectangle.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionReadMe">
   <property name="text">
    <string>ReadMe</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameEditor</class>
   <extends>QWidget</extends>
   <header>frameeditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="iconResources.qrc"/>
 </resources>
 <connections/>
</ui>
